src/testdir/test_search_stat.vim

```diff
@@ -492,6 +492,7 @@ endfunc
 func Test_search_stat_option()
   " Asan causes wrong results, because the search times out
   CheckNotAsan
+  let g:test_is_flaky = 1
```

Comment:
This test still sometimes fails (e.g. <https://github.com/vim/vim/actions/runs/16919563215/job/47941992737>), so mark it as flaky.